src/Agoda.Analyzers/AgodaCustom/CustomRulesResources.resx

```diff
@@ -186,4 +186,10 @@
   <data name="AG0026Title" xml:space="preserve">
     <value>Use only CSS Selectors to find elements in Selenium tests</value>
   </data>
+  <data name="AG0019Title" xml:space="preserve">
+    <value>Prevent use of System.Runtime.CompilerServices.InternalsVisibleTo attribute</value>
```

Comment:
Maybe the title should say 
> Do not use

rather than 
> Prevent use

Reply:
Ok.